lmp/build-sdk-container.sh

```diff
@@ -18,11 +16,15 @@ for i in `seq 12` ; do
 done
 
 container="hub.foundries.io/lmp-sdk"
-tagged="${container}:${TAG}"
-latest="${container}:latest"
```

Comment:
I think you'll wind up needing a block like:
 https://github.com/foundriesio/ci-scripts/blob/master/lmp/jobserv.yml#L18-L24

for changes to the "next" branch. Seems like the easiest way to control that will be via an environment variable. e.g.
```
- name: lmp-sdk-next
        host-tag: amd64
        container: foundries/dind-ci:19.03.9_b38f166
        privileged: true
        params:
           LATEST=next
        script-repo:
          name: fio
          path: lmp/build-sdk-container.sh
 ```

And then you could have this script take a variable for "latest". e.g:
```
LATEST=${LATEST:latest}
``

Reply:
Well! With the argument it wouldn't work. thanks for the tip.
Also added the block on the main build job:
```
- name: lmp-sdk-next
        host-tag: amd64
        container: foundries/dind-ci:19.03.9_b38f166
        privileged: true
        params:
           LATEST=next
        script-repo:
          name: fio
          path: lmp/build-sdk-container.sh
```